### PR TITLE
TRIVIAL: Update changelog for 7.3.0

### PR DESCRIPTION
**Pull request by xMort**
---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/5978/ : SUCCESS in 11m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/1604/ : SUCCESS in 4s

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,20 @@ The REST API versions in the table are just for your information as the values a
 
 - We ask developers to consider using the Headline component instead of the KPI component. The KPI component may be eventually marked as deprecated in one of the next major versions.
 
+## 7.3.0
+
+### Added
+
+- Measure value filter
+
+### Fixed
```
Comment (evgeniabogranskaya) [inline CHANGELOG.md]: - Attribute sort aggregation is now preserved in pivot tables.
- Microsoft Internet Explorer and Microsoft Edge no longer show two "X" icons in the search input field.
- Invalid autoprefixer directives were fixed in the main stylesheet.
- react-intl was upgraded to the latest version.
- @gooddata/numberjs was upgraded to the latest version.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6007/ : FAILURE in 30m 55s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/1637/ : SUCCESS in 4s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6008/ : SUCCESS in 17m 34s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/1638/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3096/ : SUCCESS in 15m 35s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6009/ : SUCCESS in 7m 33s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1442/ : SUCCESS in 29s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2007/ : SUCCESS in 19m 58s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3097/ : UNSTABLE in 16m 20s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/6010/ : SUCCESS in 8m 14s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1443/ : SUCCESS in 1m 05s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2008/ : SUCCESS in 19m 50s

**Pull request merged**